index.html

```diff
@@ -1,24 +1,234 @@
+                            ,in korea.
+                        </p>
+                        <p class="date">2022.10.17(THUR) ~ 30(FRI)</p>
+                        <p class="location">@ National Museum of Korea, Art Center Nabi and more</p>
+                    </div>
+                </div>
+            </div>
+        </section>
+
+        <section>
+            <div class="cover-darker">
+                <div class="container">
+                    <div class="main-program">
+                        <div>
+                            <h2>main program</h2>
+                            <p class="underline"></p>
+                        </div>
+                        <div class="cards">
+                            <div class="card">
+                                <i class="fa fa-podcast fa-3x"
+                                    aria-hidden="true"></i>
+                                <h3>lecture</h3>
+                                <p>Diam justo et amet sadipscing accusam duo et.
+                                    Sea ipsum lorem est sadipscing ipsum est
+                                    duo.</p>
+                            </div>
+                            <div class="card">
+                                <i class="fa fa-shopping-bag fa-3x"
+                                    aria-hidden="true"></i>
+                                <h3>CC exhibition</h3>
+                                <p>Seemed and charms cheer from for he, to true
+                                    one amiss maidens whose mine below dares..</p>
+                            </div>
+                            <div class="card">
+                                <i class="fa fa-forumbee fa-3x"
+                                    aria-hidden="true"></i>
+                                <h3>forum</h3>
+                                <p>Und folgenden michjenem folgenden ihr euch
+                                    dunst ist fühlt busen, fühl die.</p>
+                            </div>
+                            <div class="card">
+                                <i class="fa fa-building fa-3x"
+                                    aria-hidden="true"></i>
+                                <h3>workshop</h3>
+                                <p>Fyodumtul nym hioll de um yg symeonnok
+                                    therthetyk vh ualallal uiraga werethul
+                                    therthetyk. Urodum.</p>
+                            </div>
+                            <div class="card">
+                                <i class="fa fa-free-code-camp fa-3x"
+                                    aria-hidden="true"></i>
+                                <h3>CC ignite</h3>
+                                <p>Kiun sakon tien-cxi metitan veloj la povis
+                                    sendube gastoj sian sed, ridetis en knabo
+                                    pri la sxipo. Tiam tiom bestoj.</p>
+                            </div>
```

Comment:
- For better semantics and to improve your site's accessibility, I'd suggest you use an `article` tag for each of your tags, please fix this 👍 👍
- In your mobile version, the content of the cards aren't properly aligned, please fix this issues 🙏 🙏 

Your project(Mobile version):
<img width="144" alt="pius main prog mbile" src="https://user-images.githubusercontent.com/73607512/198259238-9458fe6d-69b5-4c07-8b39-9527bebd9641.PNG">

Design Template:
<img width="285" alt="mobile mainprograme behance" src="https://user-images.githubusercontent.com/73607512/198259331-df36fbc0-4e3c-4e12-badf-b7b4a7a4604b.PNG">